subjects/localhost/README.md

```diff
@@ -0,0 +1,162 @@
+## Localhost
+
+The Hypertext Transfer Protocol (HTTP) is an application layer protocol in the Internet protocol suite model for distributed, collaborative, hypermedia information systems.
+HTTP is the foundation of data communication for the World Wide Web. Hypertext documents include hyperlinks to other resources that the user can easily access, for example, by a mouse click or tap on the screen in a web browser.
+
+### Instructions
+
+#### Server
+
+The project can be written in one of these languages [Go,C++,C...]
+
+```console
+$ ./localhost [CONFIG FILE]
+Lisning on  [IP]:[PORT]
+Lisning on  [IP]:[PORT2]
+```
+- Your server must not crash for any reason.
+- Any request to your server should never hang forever.
+- Your server can listen on multiple servers and ports at the same time without any conflict 
+- You must use the single process and no treads.
+- You must use `exec` functions only in `CGI` part
+- Your server must be compatible with the last version of your chosen browser 
+- You can use `NGINX` to compare headers and answer behaviors with your server
+- You must manage at least [`GET`, `POST`, `DELETE`] methods.
+- You must handle customization errors **(Binding error, default page error...)**
+- You must create error page at least for [400,311,403,404,405,413]
```

Comment:
May be clarified by saying "... create default error page**s** for at least the following error codes..."